.gitignore

```diff
@@ -0,0 +1,2 @@
+# Ansible
+ansible/ansible-deps-cache
```

Comment:
Almost all files are under `ansible/`. Do we need that subdirectory? If not, please move all files there one directory up.

Reply:
Thank you. Fixed in #2, as this one was already merged at the time of your review.